.travis.yml

```diff
@@ -39,25 +40,26 @@ matrix:
     - rust: nightly
   include:
   - rust: nightly-2018-08-17
-    env: CLIPPY_AND_COMPILE_TESTS=YESPLEASE
+    if: os = linux
+    name: "Clippy && Compile tests"
```

Comment:
Is this new ?

Reply:
Don't know, just saw it [here](https://www.reddit.com/r/rust/comments/9ndo5k/travis_ci_now_supports_testing_windows_includes/e7livzr/)

Reply:
Nice